nil/services/cometa/compilation.go

```diff
@@ -138,7 +138,33 @@ func Compile(input *CompilerTask) (*ContractData, error) {
 func CreateContractData(input *CompilerJsonInput, outputJson *CompilerJsonOutput) (*ContractData, error) {
 	contractData := &ContractData{}
 
-	contractData.SourceFilesList = make([]string, len(input.Sources)+1)
+	numContracts := len(input.Sources)
+
+	var contractDescr *CompilerOutputContract
+	for _, v := range outputJson.Contracts {
+		if len(v) > 1 {
+			return nil, errors.New("expected exactly one contract in compilation output")
+		}
+		if len(v) != 0 {
+			for _, c := range v {
+				contractDescr = &c
+				break
+			}
+			break
+		}
```

Comment:
Why not something like?
```suggestion
		if len(v) != 1 {
			return nil, errors.New("expected exactly one contract in compilation output")
		}
		contractDescr = &v[0]
```